### ISSUE TITLE
Support threadsafe: true

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Dragon-Born/google-api-python-client#136]
```
The package doesn't support working with threadsafe: true, this cause the 
entire app not being able to work with multiple threads.
```

Original issue reported on code.google.com by `erlichmen` on 24 May 2012 at 7:31

**Comment by GoogleCodeExporter** [comment]
```
Objects from the library aren't threadsafe, so that means you just have to make 
sure you use one httplib2.Http() object per thread. The only other outstanding 
problem is the way decorators are handled, which is in the app.yaml file, which 
will change when this CL lands:

  http://codereview.appspot.com/6441056/
```

Original comment by `jcgregorio@google.com` on 1 Aug 2012 at 6:36
- Changed state: **Accepted**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 9c4d5eb9cb82.
```

Original comment by `jcgregorio@google.com` on 3 Aug 2012 at 8:18
- Changed state: **Fixed**
